include/crm/fencing/internal.h

```diff
@@ -105,7 +105,10 @@ void stonith__device_parameter_flags(uint32_t *device_flags,
                                      const char *device_name,
                                      xmlNode *metadata);
 
-#  define ST_LEVEL_MAX 10
+/* Only 1-9 is allowed for fencing topology levels,
+ * however, 0-10 has been used internally.
+ */
+#  define ST_LEVEL_COUNT 10
```

Comment:
Let's use a double underbar to indicate this is internal (and same for the MIN/MAX)